.github/workflows/test_asterinas.yml

```diff
@@ -86,3 +86,53 @@ jobs:
       - name: General Test (Linux EFI Handover Boot Protocol)
         id: test_linux
         run: make run AUTO_TEST=test ENABLE_KVM=0 BOOT_PROTOCOL=linux-efi-handover64 RELEASE=1
+
+  integration-test-with-kvm-enabled:
```

Comment:
We need to balance between the coverage and cost of testing. Ideally, we want to test every combination of configurations such as KVM enabled/disabled, release/debug builds, and boot protocols. But there are too many combinations. Running the entire integration test with KVM enabled again after doing it already with KVM disabled is not cost effective. I suggest running all tests with KVM enabled by default and only disabling KVM for a selected set of tests. One strategy is to disable KVM only when running "Syscall Test at Exfat".